### ISSUE TITLE
Missing doctrine/collections dependency in 6.4.20

### ISSUE THREAD

**Issue by breaker92** [shopware/platform#3227]
### PHP Version

8.1

### Shopware Version

6.4.20.2

### Expected behaviour

The class `Shopware\Core\Content\ImportExport\ImportExport` uses the class `Doctrine\Common\Collections\ArrayCollection`, which comes with the package `doctrine/collections`.

But `doctrine/collections` isn't a prod dependency of `shopware/core`, neither directly nor indirectly.

### Actual behaviour

If you only install the no-dev dependencies, you won't get `doctrine/collections` and the ImportExport will crash with an class not found error.

### How to reproduce

Require `shopware/core` and run `composer install --no-dev`

**Comment by maximilian-schwarz** [comment]
Had the same issue

**Comment by mchh** [comment]
We can also reproduce this issue on production shops. Import doesn't work.

**Comment by cngJo** [comment]
We had the same issue and @BeerOnTheBeach and I went into that rabbit hole :D

It seems like `doctrine/collections` was an indirect dependency of `shopware/core > equeue/dbal > doctrine/persistence > doctrine/collections`.

`doctrine/persistence` dropped the dependency of `doctrine/collections` with version 3.0.3 with https://github.com/doctrine/persistence/pull/302.

We solved it with `composer require doctrine/collections` at version 1.8.x in out project.

But would be great to have this fixed in Shopware. What would be the best way to create a Pull Request for this? Just point it at the `6.4.20.0` branch? Would be happy to provide a PR for this :)
